src/migrations/helpers/get-deployed-contracts.ts

```diff
@@ -51,5 +71,14 @@ export async function getDeployedContracts(): Promise<DeployedContracts> {
     safetyModuleNewImpl: new SafetyModuleV2__factory(deployer).attach(deployedAddresses.safetyModuleNewImpl),
     safetyModuleRecovery: new SM2Recovery__factory(deployer).attach(deployedAddresses.safetyModuleRecovery),
     safetyModuleRecoveryProxyAdmin: new ProxyAdmin__factory(deployer).attach(deployedAddresses.safetyModuleRecoveryProxyAdmin),
+    liquidityStaking: new LiquidityStakingV1__factory(deployer).attach(deployedAddresses.liquidityStaking),
+    liquidityStakingProxyAdmin: new ProxyAdmin__factory(deployer).attach(deployedAddresses.liquidityStakingProxyAdmin),
+    merkleDistributor: new MerkleDistributorV1__factory(deployer).attach(deployedAddresses.merkleDistributor),
+    merkleDistributorProxyAdmin: new ProxyAdmin__factory(deployer).attach(deployedAddresses.merkleDistributorProxyAdmin),
+    starkProxies: deployedAddresses.starkProxies.map((s) => new StarkProxyV1__factory(deployer).attach(s)),
+    starkProxyProxyAdmins: deployedAddresses.starkProxyProxyAdmins.map((s) => new ProxyAdmin__factory(deployer).attach(s)),
+    dydxCollateralToken: new MintableERC20__factory(deployer).attach(deployedAddresses.dydxCollateralToken),
+    starkPerpetual: new MockStarkPerpetual__factory(deployer).attach(deployedAddresses.starkPerpetual),
```

Comment:
This is just a note -- you don't necessarily have to make changes. But there's something logically weird here if `MintableERC20` and `MockStarkPerpetual` are being used outside of the `test/` directory.

Consider using `IStarkPerpetual` instead of `MockStarkPerpetual` unless functions from the mock interface are needed. Consider adding a TODO to clean this up.